app/pubspec.yaml

```diff
@@ -86,7 +86,7 @@ dependencies:
   geolocator:
   version: ^3.0.2
   webview_flutter: ^4.8.0
-  flutter_sound: ^9.10.0
+  flutter_sound: 9.10.0
```

Comment:
This is causing issues with latest version (https://github.com/Canardoux/flutter_sound_web/blame/b64335b030592f8b36c4459ce2803b3ad29a0ebb/lib/flutter_sound_media_player_web.dart#L83-L86) which works on latest version of `web` dependency, but we're using `file_picker` which uses `0.5.1` version. I don't want update `file_picker` not sure what breaking changes they have. So I had to pin the version.